cmd/bootstrap/runner.go

```diff
@@ -461,23 +454,15 @@ func (r *runner) installChartMuseum(ctx context.Context, appTest apptest.Interfa
 	{
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating %#q app cr", chartMuseumName))
 
-		valuesYAML := `persistence:
-  enabled: "true"
-serviceAccount:
-  name: "chartmuseum"
-  create: "true"
-env:
-  open:
-    DISABLE_API: false`
-
 		apps := []apptest.App{
 			{
-				CatalogName: helmStableCatalogName,
-				CatalogURL:  helmStableCatalogStorageURL,
-				Name:        chartMuseumName,
-				Namespace:   namespace,
-				ValuesYAML:  valuesYAML,
-				Version:     chartMuseumVersion,
+				CatalogName:   helmStableCatalogName,
+				CatalogURL:    helmStableCatalogStorageURL,
+				Name:          chartMuseumName,
+				Namespace:     namespace,
+				ValuesYAML:    chartMuseumValuesYAML,
+				Version:       chartMuseumVersion,
+				WaitForDeploy: r.flag.Wait,
```

Comment:
This is the actual change. The best way to be sure everything is up and running is to deploy something.

For now it still uses chartmuseum. When we switch to another web server for app build suite support I'd still like to deploy it via an app CR.